credsweeper/utils/util.py

```diff
@@ -246,6 +246,23 @@ def is_zip(data: bytes) -> bool:
                     return False
         return False
 
+    @staticmethod
+    def is_tar(data: bytes) -> bool:
+        """According https://en.wikipedia.org/wiki/Bzip2"""
+        if isinstance(data, bytes) and 262 <= len(data):
+            if 0x75 == data[257] and 0x73 == data[258] and 0x74 == data[259] and 0x61 == data[260] \
+                    and 0x72 == data[261]:
+                return True
+        return False
+
+    @staticmethod
+    def is_bzip2(data: bytes) -> bool:
```

Comment:
What are you going to do if you are wrong in the decision that this is an archive?
BZh is a printable and thus may be present in some texts or binary data.
It seems to me that it is too simple format decision function, it should be extended to include some other fields, e.g. lengths, dir construct, etc.

Reply:
The simple detection is applied due some files are *.tar.bz2. 
Yes, if BZh is placed at start of a file - it will be parsed as the archive but exception raised.